### PR TITLE
Remove customer returns

### PR DESCRIPTION
**Pull request by mbj**

### CODE REVIEW

**Comment by mbj** [comment]
@dkubb Ready for conceptual review. I propose to break this up into 3 stages:

1. Backend.
2. API.
3. Core.

In our main app the entire subsystem is unused.

File: api/spec/controllers/spree/api/orders_controller_spec.rb [hunk 1]
```diff
@@ -106,14 +106,12 @@ module Spree
...
-            let(:order_state) { order_state }
+        context "order is in the 'completed' state" do
+          let(:order_state) { 'completed' }
 
-            it "returns no content" do
-              expect(subject.status).to eq 204
-              expect(subject.body).to be_blank
-            end
+          it "returns no content" do
+            expect(subject.status).to eq 204
```
Comment (dkubb) [inline api/spec/controllers/spree/api/orders_controller_spec.rb]: What about using `be(204)` ?

File: api/spec/controllers/spree/api/orders_controller_spec.rb [hunk 2]
```diff
@@ -106,14 +106,12 @@ module Spree
...
+            expect(subject.body).to be_blank
```
Comment (dkubb) [inline api/spec/controllers/spree/api/orders_controller_spec.rb]: Is the expected body `nil` or `''`. Maybe this could be made more precise?

**Comment by mbj** [comment]
Closing this PR in favor of the extractions.

**Pull request closed without merging**